packages/tw-shimmer/src/index.css

```diff
@@ -15,11 +24,37 @@
   --shimmer-color: black;
   --shimmer-spread: 6ch;
   --shimmer-angle: 90deg;
+  /* Eased gradient: sine ease-in-out for smooth falloff at both center and edges */
+  --shimmer-highlight: color-mix(
```

Comment:
highlight should be same as shimmer color? unsure why it should be 90% of the user defined color
make it default to 90% of current color & black if you want, sure